packages/safe-tools-client/app/services/network.ts

```diff
@@ -47,14 +50,24 @@ export default class NetworkService extends Service {
     };
   }
 
-  get supportedList() {
-    return supportedChainsArray
-      .map((networkSymbol) => ({
-        name: getConstantByNetwork('name', networkSymbol),
-        chainId: getConstantByNetwork('chainId', networkSymbol),
-        symbol: networkSymbol,
-      }))
-      .filter(({ name }) => name !== this.name);
+  get supportedNetworks() {
+    // Initially return all the scheduler-capable networks the SDK supports, and then
+    // narrow it down after retrieving the networks supported by the hub.
```

Comment:
Why is this step needed? Can we just use `remoteConfig.web3.schedulerNetworks` without using it to narrow down supportedChainsArray?

Reply:
I did it this way to avoid a loading state. We start with all the options (available synchronously) and then narrow down to what the current hub supports.